### DISCUSSION TITLE
Previous page function of the browser problem

### DISCUSSION THREAD

**Discussion by kayinortin** [igordanchenko/yet-another-react-lightbox#214] [Q&A] [answered]
Hello, my project is next.js version 14. When I click on the lightbox, it can start smoothly, but if I directly click on the previous page function of the browser, it can jump back to the previous page smoothly, and the lightbox will be closed because I return to the previous page, but because I am not pressing Close on the original page button, making it impossible to click on any object on the page after the jump.

Could you share how to handle this incident?

![1](https://github.com/igordanchenko/yet-another-react-lightbox/assets/48908211/36ad9800-1947-47f9-a83f-ea7fa98c10b6)

![2](https://github.com/igordanchenko/yet-another-react-lightbox/assets/48908211/d2b19d76-bac8-46b1-ad56-88d68d1e2781)

https://codepen.io/Edison-Li-the-selector/pen/oNmayrb

**Comment by igordanchenko** [comment, marked as answer]
There was a [bug](https://github.com/igordanchenko/yet-another-react-lightbox/issues/209) affecting versions 3.15.0 through 3.15.3. Upgrading to v3.15.4+ will resolve this issue.

**Reply by kayinortin** [reply]
thank you
